assets/components/src/with-wizard-screen/index.js

```diff
@@ -9,12 +9,17 @@ import { Component, Fragment } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
```

Comment:
On mobile (iPhone XS/Mobile Safari), sometimes the two buttons don't fit horizontally and end up with a broken layout:

<img width="1595" alt="Screen Shot 2019-12-11 at 11 07 16 PM" src="https://user-images.githubusercontent.com/1477002/70681929-2e2cfc80-1c6b-11ea-845d-ebd399f12a4f.png">

Reply:
Looks like the expected behaviour to me. How is that broken?

Reply:
I wasn't sure if you wanted them to lay out horizontally. If this is expected, ignore the comment.